### PR TITLE
Fix cw1-subkeys helper.ts and point to heldernet

### PR DESCRIPTION
**Pull request by orkunkl**
This change is required for coralnet to heldernet update in docs.

### CODE REVIEW

File: contracts/cw1-subkeys/helpers.ts [hunk 1]
```diff
@@ -20,24 +20,29 @@ interface Options {
...
+  gasPrice:  GasPrice.fromString("0.025ucosm"),
+  bech32prefix: 'cosmos',
+  feeToken: 'ucosm',
+  faucetUrl: 'https://faucet.heldernet.cosmwasm.com/credit',
   hdPath: makeCosmoshubPath(0),
-  defaultKeyFile: path.join(process.env.HOME, ".coral.key"),
+  defaultKeyFile: path.join(process.env.HOME, ".heldernet.key"),
+  gasLimits: {
+    upload: 1500000,
+    init: 600000,
```
Comment (ethanfrey) [inline contracts/cw1-subkeys/helpers.ts:42]: I think these can be brought down a lot. We know what the contract does.

Anyway, low priortiy, you can merge without this. Just want to mark it as a TODO

**Review by ethanfrey — APPROVED** [review]
Looks good. Same general comment as #201 

Only major change needed is setting a  cosmjs version in the header.

**Comment by orkunkl** [comment]
Same question goes for this PR: https://github.com/CosmWasm/cosmwasm-plus/pull/201#issuecomment-749413463

**Comment by ethanfrey** [comment]
We used v0.10.4, [see here](https://github.com/CosmWasm/cosmwasm-plus/blob/fd542d229c17b21079ce2c325ab92cc159d312f9/.circleci/config.yml#L867)

Nice to point this out. v0.4.0 will be built with v0.10.7

**Pull request merged**